### DISCUSSION TITLE
Referencing docs between multi-doc instances not possible when using .md/.mdx in URL?

### DISCUSSION THREAD

**Discussion by Dannyholt** [facebook/docusaurus#4388] [Q&A] [answered]
I have a few multi-doc instances e.g. `/faq`, `/api`, `/theming`. I have a doc in the `/faq` docs instance which i'm trying to reference on another docs instance e.g. `/api`.

When I use `[hello](/faq/hello.md)` this causes a 404, but if I remove the `.md` i.e. `[hello](/faq/hello)` this works. So my question is this intended or should using the `.md/.mdx` also work?

Also, how would this impact when using versioning. I'm asking because there are quite a few links in our docs that reference other docs which we're in the process of migrating to separate doc instances to make this easier to manage.

**Comment by slorber** [comment, marked as answer]
Hi,

If you want to use md file paths you should use relative paths otherwise those paths will be resolved from the root of your FS (not 100% sure,  may be wrong)

So the correct relative path may probably be something like: `[hello](../faq/hello.md)`.
Using relative paths is important for versioning because it ensure that `/1.0/faq` links to `/1.0/api` instead of linking to `/api`.

Also, one plugin instance is not aware of the route paths and md files of the other plugin instances so it is not possible to link this way from one plugin to another. 


If you have ios 1.0 + 2.0, and android 2.0 + 3.0. When android 3.0 use a link like `[ios](/ios-intro.md)`, it is impossible for Docusaurus to know if you want to link to `/ios/1.0/intro/` or `/ios/2.0/intro/`, this is why for linking across plugins you'd rather be explicit and use the full absolute paths
